hastory/app/Main.hs

```diff
@@ -1,5 +1,6 @@
 module Main where
 
+import Prelude (IO)
```

Comment:
This is quite silly.
Better remove
```
    default-extensions: NoImplicitPrelude
```
instead.

Reply:
I'll certainly remove in the final version - at the moment I am trying to keep my ghcid output as clean as possible and it complains unless the `IO` type constructor is in scope.